app/src/main/kotlin/com/wire/android/ui/common/FloatingActionButton.kt

```diff
@@ -1,25 +1,32 @@
 package com.wire.android.ui.common
 
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
 import androidx.compose.material3.ExtendedFloatingActionButton
 import androidx.compose.material3.Icon
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
-import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.filled.Add
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import com.wire.android.ui.theme.wireTypography
 
 @Composable
 fun FloatingActionButton(
     text: String,
-    onClick: () -> Unit,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    icon: @Composable (() -> Unit)? = null,
```

Comment:
what is the advantage of using Composable lambda here? Also I think it would be more readable to just assign a default value instead of making it nullable

Reply:
If the icon should be optional then we don't want a default value and current solution with lambda and null is fine. The question is: should the icon really be optional or is it always on the FAB?

Reply:
True true it should be optional but if you read further in the code it has a default value anyway